galasa-ui/src/components/test-runs/TestRunDetails.tsx

```diff
@@ -96,11 +107,14 @@ const TestRunDetails = ({ runId, runDetailsPromise, runLogPromise, runArtifactsP
   if (isError) {
     return <ErrorPage />;
   }
-  
+
+  // Build the test runs route with the saved query string
+  const testRunsRoute = savedQuery ? `${TEST_RUNS.route}?${savedQuery}` : TEST_RUNS.route;
+  const testRunsBreadCrumb = { ...TEST_RUNS, route: testRunsRoute };
```

Comment:
I think this breadcrumb will still take you back to the "timeframe" tab, rather than the "results" tab. I'll raise a separate issue for that!